stable/influxdb/templates/deployment.yaml

```diff
@@ -50,13 +50,13 @@ spec:
         {{- end }}
         livenessProbe:
           httpGet:
-            path: /ping
+            path: {{ .Values.deployment.probePath }}
             port: api
           initialDelaySeconds: 30
```

Comment:
Should we make replace teh hard code to {{  .Values.livenessProbe.initialDelaySeconds | default 30 }}